addons/account/models/account_journal.py

```diff
@@ -571,7 +576,7 @@ def _alias_get_creation_values(self):
             values['alias_name'] = self._alias_prepare_alias_name(self.alias_name, self.name, self.code, self.type, self.company_id)
             values['alias_defaults'] = defaults = literal_eval(self.alias_defaults or "{}")
             defaults['company_id'] = self.company_id.id
-            defaults['move_type'] = 'in_invoice' if self.type == 'purchase' else 'out_invoice'
+            defaults['move_type'] = 'in_invoice' if self.type == 'purchase' else 'out_invoice' if self.type == 'sale' else 'entry'
```

Comment:
Don't make a ternary conditionception
```suggestion
            defaults['move_type'] = {
                'purchase': 'in_invoice',
                'sale': 'out_invoice',
            }.get(self.type, 'entry')
```